inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/metrics/SortSdkPrometheusMetricListener.java

```diff
@@ -41,7 +41,7 @@ public SortSdkPrometheusMetricListener(String sortTaskId) {
 
         final MBeanServer mbs = ManagementFactory.getPlatformMBeanServer();
         StringBuilder beanName = new StringBuilder();
-        beanName.append(JMX_DOMAIN).append(DOMAIN_SEPARATOR).append("type=SortSdkCounter");
+        beanName.append(JMX_DOMAIN).append(DOMAIN_SEPARATOR).append("type=SortSdk-").append(sortTaskId);
```

Comment:
It is better to add a new property:name for sortTaskId.
![image](https://user-images.githubusercontent.com/8925507/162357238-21a960fc-da52-43d7-b58c-70c777086126.png)

Reply:
👌，i will change it like this